src/navigation_controller.ts

```diff
@@ -53,23 +62,24 @@ export class NavigationController {
   }
 
   /**
-   * Adds methods to core Blockly components that allows them to handle keyboard
-   * shortcuts when in keyboard navigation mode.
+   * Monkeypatches core Blockly components to add methods that allow
```

Comment:
Historical note: this used to also monkeypatch some fields. I deleted that during cleanup, leaving only the toolbox monkeypatching.

Reply:
Thanks for this info.